docs/magento1/headless-payments.mdx

```diff
@@ -83,3 +95,32 @@ code and `{class_name}` by the class names you've just created
 - `returnAction(String $action, [String] $additionalData): Boolean` Callback
   after payment, for example, you can add your custom code here for retrieve
   customer cart after cancel payment
+
+:::tip REMINDER
+
+Don't forget to override or update `theme/pages/Checkout/checkoutFlowOf.js` to
```

Comment:
**nit-pick:** We can include the full path in the title of the diff code block which makes the text easier to read, see next comment

```suggestion
Don't forget to override or update `checkoutFlowOf.js` to
```

Reply:
OH, I didn't knew that.

Thanks for the tip!